x-pack/plugins/infra/public/alerting/log_threshold/components/alert_details_app_section/index.tsx

```diff
@@ -50,12 +46,35 @@ const AlertDetailsAppSection = ({
   alert,
   setAlertSummaryFields,
 }: AlertDetailsAppSectionProps) => {
-  const [selectedSeries, setSelectedSeries] = useState<string>('');
-  const { uiSettings, observability } = useKibanaContextForPlugin().services;
-  const { euiTheme } = useEuiTheme();
+  const { observability, data, http } = useKibanaContextForPlugin().services;
   const theme = useTheme();
   const timeRange = getPaddedAlertTimeRange(alert.fields[ALERT_START]!, alert.fields[ALERT_END]);
   const alertEnd = alert.fields[ALERT_END] ? moment(alert.fields[ALERT_END]).valueOf() : undefined;
+  const interval = `${rule.params.timeSize}${rule.params.timeUnit}`;
+  const thresholdFill = convertComparatorToFill(rule.params.count.comparator);
+  const filter = rule.params.groupBy
+    ? rule.params.groupBy
+        .map((field) => {
+          const value = get(
+            alert.fields[ALERT_CONTEXT],
+            ['groupByKeys', ...field.split('.')],
+            null
+          );
+          return value ? `${field} : "${value}"` : null;
+        })
+        .filter(identity)
+        .join(' AND ')
+    : '';
+
+  const logViews = useMemo(
+    () => new LogViewsClient(data.dataViews, http, data.search.search, defaultLogViewsStaticConfig),
+    [data.dataViews, data.search.search, http]
+  );
```

Comment:
You can use `logViews` from services: https://github.com/elastic/kibana/blob/39ddf4d96b2b05faf4b0d67108956d1292420706/x-pack/plugins/infra/public/alerting/log_threshold/components/alert_details_app_section/components/explain_log_rate_spike.tsx#L54

Reply:
FYI, this will be changed in Marco's [PR](https://github.com/elastic/kibana/pull/159279):

https://github.com/elastic/kibana/pull/159279/files#diff-ef33ba02a14547c301fd20bbb14de418dd0a13d6d5124178ea9f333eaa9a1facL54

Reply:
Hey there, I'll give another pass on the service usage after rebasing the last changes and update this usage for you before merging 👌